### ISSUE TITLE
cordova-serve-ios does not generate the build/emulator folder

### ISSUE THREAD

**Issue by vesper8** [m0dch3n/vue-cli-plugin-cordova#124]
I'm really confused because I could swear this used to work. I updated a lot of dependencies lately and now I'm noticing that when I run `cordova-serve-ios` after running `cordova-build-only-www-ios`, it does not update the files inside the `src-cordova/platforms/ios/build/emulator` folder and what ends up launching on my IOS Simulator is an old version

If I delete the `emulator` folder manually, it does not get recreated

The only way I've found to create this folder, and correctly open the Simulator with the latest version is to run `cordova emulate ios` from inside the `src-cordova` folder

So I wonder then what is the point of the `cordova-serve-ios` command now.. I thought this was meant to test out in a simulator or alternatively a USB connected device

**Comment by codecat** [comment]
I have a similar problem with `cordova-serve-ios` where live reload doesn't properly reload the html on the device, even if I press Cmd+R in the Safari web developer window. This seems to have started after installing ios-cordova 6.x, as it worked fine on 5.1.1.

The difference is that 5.1.1 is using `UIWebView` (which Apple doesn't support anymore) and 6.x is using `WKWebView` (which is required for submitting apps to the app store now).

**Comment by yokins** [comment]
@codecat I have the same problem, do you have any solution?

**Comment by codecat** [comment]
I have not found a solution, however it seems to have been happening less often with the ionic webview.

I just restart the app whenever it starts behaving strangely and that fixes it for a while.

**Comment by yokins** [comment]
yes, only clean safari cache or restart app, then the page will refresh
